.eslintrc.js

```diff
@@ -15,7 +15,6 @@ module.exports = {
     "plugin:jsx-a11y/recommended",
     "airbnb-typescript",
     "plugin:sonarjs/recommended",
-    "plugin:jest-playwright/recommended",
```

Comment:
As far as I can tell, there aren't any playwright tests.

Reply:
Are we adding any today?

Reply:
It's not mentioned in the spreadsheet.

Also, Playwright itself isn't in this repo. This is just an eslint plugin

Reply:
That sounds good! With Chromatics, maybe there's no immediate need to use Playwright. But if we do, we can add back later 👍